stance/main.py

```diff
@@ -1,23 +1,25 @@
 import argparse
+import logging
+import pickle
 from etl import template_fit, etl
 from motion.squat import Squat
 from predict import score, overlay
-import pickle
 
-if __name__ == "__main__":
+
+def main():
```

Comment:
(((Also the pickle files)))

Reply:
Yeah so that's a fundamental problem with the way we do things. Rather than enforcing that the user has to be in a certain directory, our code should be independent of that.